modules/gapi/include/opencv2/gapi/streaming/gstreamer/gstreamerpipeline.hpp

```diff
@@ -19,15 +19,18 @@ namespace gapi {
 namespace wip {
 namespace gst {
 
-class GAPI_EXPORTS GStreamerPipeline
+class GAPI_EXPORTS_W GStreamerPipeline
 {
 public:
     class Priv;
 
-    explicit GStreamerPipeline(const std::string& pipeline);
+    GAPI_WRAP explicit GStreamerPipeline(const std::string& pipeline);
     IStreamSource::Ptr getStreamingSource(const std::string& appsinkName,
                                           const GStreamerSource::OutputType outputType =
                                               GStreamerSource::OutputType::MAT);
+    GAPI_WRAP cv::Ptr<cv::gapi::wip::IStreamSource> get_streaming_source(
```

Comment:
Why don't you wrap `getStreamingSource` instead of implementing the new one?

Reply:
the new method has different output type;

I didn't manage to make `IStreamSource::Ptr` working in Python

also, as I understand, this is why we have `make_capture_src` and `make_gst_src` "Python-only" functions instead of just wrapping already existing `make_src`, besides it is a template